js/src/setup-mc/setup-stepper/index.scss

```diff
@@ -2,15 +2,15 @@
 	.woocommerce-stepper__steps {
 		align-items: center;
 		justify-content: center;
-		height: 64px;
-		background-color: #fff;
-		box-shadow: inset 0 -1px 0 #ccc;
+		height: $grid-unit-40 * 2;
```

Comment:
❔ Same as https://github.com/woocommerce/google-listings-and-ads/pull/1578#discussion_r920843375.